andela/Ah-frontend-guardians
# chore (setupSCSS): setup SCSS

#### What does this PR do?
This PR sets up SCSS.

#### Description of Task to be completed?
The task involves setting up SCSS as a most commonly used syntax for SASS.
SCSS stands for "Sassy CSS"

- install sass-loader

- install css-loader

- install node-sass

- install extract-text-webpack-plugin

NOTE: All the installations involve the word; --save-dev. This helps to ensure that the installed project dependencies get saved among the dev-dependencies.

- Run npm install on your machine to install the package.json file

#### How should this be manually tested?

- Manually check out for the installed project dependencies among the dev-dependencies in package.json file

- Run the command below;
npm run build


#### What are the relevant pivotal tracker stories?
#162685621
